### PR TITLE
Graph feature implementation

### PR DESCRIPTION
**Pull request by mehedihasan88**

### CODE REVIEW

File: TeamTracker/app/src/main/java/com/example/teamtracker/util/DateTimeUtil.java [hunk 1]
```diff
@@ -30,4 +31,42 @@ public static String getDateFromEpoch(Long epoch) {
...
+                break;
+            case "Thursday":
+                day = 5;
+                break;
+            case "Friday":
+                day = 6;
+                break;
+        }
+        return day;
+    }
```
Comment (mehedihasan88) [inline TeamTracker/app/src/main/java/com/example/teamtracker/util/DateTimeUtil.java:77]: better implementation possible :|

File: TeamTracker/app/src/main/java/com/example/teamtracker/fragments/GraphFragment.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.teamtracker.R;
+import com.example.teamtracker.models.Project;
+
+
+public class GraphFragment extends Fragment {
+
+
```
Comment (sajjad-h) [inline TeamTracker/app/src/main/java/com/example/teamtracker/fragments/GraphFragment.java:21]: remove extra lines

File: TeamTracker/app/src/main/java/com/example/teamtracker/fragments/PersonalContributionBarchartFragment.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        String[] days = {"Sat", "Sun", "Mon", "Tue", "Wed", "Thurs",  "Fri"};
+        ArrayList<String> label = new ArrayList<>();
+        for (int day = 0; day < 7; day++)
+            label.add(days[(day + startDay) % 7]);
+        return label;
+    }
+
+    private ArrayList<Long> getGraphData(List<Task> tasks, long currentTime) {   //returns daily contribution in seconds for the last week
+        ArrayList<Long> contributionList = new ArrayList<Long>(Collections.nCopies(7, 0L));
+        Long milliSecondInAWeek = 1L*7*24*60*60*1000;
```
Comment (sajjad-h) [inline TeamTracker/app/src/main/java/com/example/teamtracker/fragments/PersonalContributionBarchartFragment.java]: private static Long MILLISECOND_IN_A_WEEK

name like this

**Review by sajjad-h — APPROVED** [review]
Good job, Mr. @mehedihasan88 👍

**Pull request merged**